web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -130,16 +130,7 @@ export class LayerDialogComponent implements OnDestroy {
     this.color = this.layer?.color || this.defaultLayerColor;
     if (!layer) {
       this.layer = this.layerService.createNewLayer();
-      const newField = this.layerService.createField(
-        FieldType.TEXT,
-        /* label= */
-        '',
-        /* required= */
-        false,
-        /* index= */
-        this.fields.size
-      );
-      this.fields = this.fields.push(newField);
+      this.createDefaultField();
```

Comment:
What's the difference between `createDefaultField()` and `addQuestion()`?

Reply:
Ah, I didn't see there was a method already!